tutorials/script/extras/conditions/1_hf_api.py

```diff
@@ -0,0 +1,142 @@
+# %% [markdown]
+"""
+# 1. HF API
+
+This module explains, how to integrate web-hosted huggingface models in your conversational services.
+"""
+
+# %pip install dff[ext,async]
+
+# %%
+import os
+from chatsky import (
+    Message,
+    RESPONSE,
+    GLOBAL,
+    TRANSITIONS,
+    LOCAL,
+)
+from chatsky import conditions as cnd
+
+from chatsky.ml.models.remote_api.hf_api_model import (
+    HFAPIModel,
+)
+from chatsky.ml import conditions as i_cnd
+from chatsky import Pipeline
+from chatsky.messengers.console import CLIMessengerInterface
+from chatsky.utils.testing.common import (
+    is_interactive_mode,
+    check_happy_path,
+    run_interactive_mode,
+)
+
+
+# %% [markdown]
+"""
+The HuggingFace inference API allows you to use any model
+on HuggingFace hub that was made publicly available by its owners.
+Pass the model address and an API key to construct the class.
+
+We are using this open source model by Obsei-AI
+to demonstrate, how custom classifiers can be easily adapted for use your script.
+"""
+
+
+# %%
+api_model = HFAPIModel(
+    model="obsei-ai/sell-buy-intent-classifier-bert-mini",
+    api_key=os.getenv("HF_API_KEY") or input("Enter HF API key:"),
+)
+
+
+# %%
+script = {
+    GLOBAL: {
+        TRANSITIONS: {
+            # We get to one of the dialog branches depending on the annotation
+            ("service", "buy", 1.2): i_cnd.has_cls_label(
+                api_model, "LABEL_1", threshold=0.95
```

Comment:
These models should be passed to pipeline (same as #376) to allow has cls label to accept model name (instead of model instance).